packages/default/scss/calendar/_theme.scss

```diff
@@ -8,6 +8,7 @@
         );
 
 
+        // TODO: do we even use this?
```

Comment:
This is used in the [jQuery Classic Calendar](https://demos.telerik.com/kendo-ui/calendar/index).